### DISCUSSION TITLE
10 pin ribbon cable for IDC missing in What to order

### DISCUSSION THREAD

**Discussion by gemarcano** [sanni/cartreader#1045] [Need help]
Am I correct in assuming that when assembling your own cartreader one also needs to buy a roll of 10pin 2.54mm pitch flat ribbon cable? The What to buy guide includes the male and female sockets for the LCD module, but from what I can tell the female ones require crimping, which implies having some ribbon cable.

If so, it might be good to add ribbon cable to the What to order wiki page.

**Comment by sakman55** [comment]
You cut the original cables that come with the LCD.  Crimp on the new females to the cut ends.

**Reply by gemarcano** [reply]
Derp, I missed the cables in the listing. Really easy to miss because they're hidden in a picture under the specifications. Good to know they're there! Thanks!
